snowshu/adapters/target_adapters/snowflake_adapter/snowflake_adapter.py

```diff
@@ -146,8 +149,8 @@ def rollback_database_creation(self, databases: Optional[set] = None):
     def create_or_replace_view(self, relation):
         pass
 
-    def create_schema_if_not_exists(self, database, schema, **kwargs):
-        database_name = self.create_database_name(database, kwargs["uuid"])
+    def create_schema_if_not_exists(self, database, schema):
+        database_name = self.create_database_name(database, self.uuid)
```

Comment:
is it available on the instance level, the uuid?

Reply:
The uuid is created on the `GraphSetRunner` object and later distributed across all `target_adapter` objects we create while running process on every graph/thread. I found this as the easiest solution to distribute the same value across all different objects instead of **kwargs that implements more complexity as we need to remember to modify also other target adapters (e.g. postgres) to accept **kwargs as well, which are not utilized afterwards.

Reply:
fixed